security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -13,6 +13,52 @@ These release notes track the development of the Compliance Operator in the {pro
+
+* Previously, the Compliance Operator would not clean up pods used to scan infrastructure when using the `debug=true` option in the scan settings. This caused pods to be left on the cluster even after deleting the Scan Setting Binding. Now, pods are always deleted when a Scan Setting Binding is deleted.(link:https://bugzilla.redhat.com/show_bug.cgi?id=2092913[*BZ#2092913*])
+
+* Previously, the Compliance Operator used an older version of `operator-sdk` that caused alerts about deprecated functionality. Now, an updated version of `operator-sdk` is included and there are no more alerts for deprecated functionality. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2098581[*BZ#2098581*])
+
+* Previously, the Compliance Operator made assumptions about machine configuration names, and as a result was unable to determine the relationship between machine configurations and kubelet configurations. Now, the Compliance Operator has improved handling of the machine configurations and is able to determine if a kubelet configuration is a subset of a machine configuration. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2102511[*BZ#2102511*])
+
+* Previously, The rule for `ocp4-cis-node-master-kubelet-enable-cert-rotation` did not properly describe success criteria. As a result, the requirements for `RotateKubeletClientCertificate` were unclear. Now, the rule for `ocp4-cis-node-master-kubelet-enable-cert-rotation` reports accurately regardless of the configuration present in the kubelet configuration file. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2105153[*BZ#2105153*])
+
+* Previously, the rule for checking idle streaming timeouts did not consider default values, resulting in inaccurate rule reporting. Now, more robust checks ensure increased accuracy in results based on default configuration values. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2105878[*BZ#2105878*])
+
+* Previously, the Compliance Operator would fail to fetch API resources when parsing machine configurations without Ignition specifications, results in `api-check-pods` to crash loop. Now, the Compliance Operator handles Machine Config Pools without Ignition specifications correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2117268[*BZ#2117268*])
+
+* Previously, rules evaluating the `modprobe` configuration would fail even after applying remediations due to a mismatch in values for the `modprobe` configuration. Now, the same values are used for the `modprobe` configuration in checks and remediations, ensuring consistent results. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2117747[*BZ#2117747*])
+
+[id="compliance-operator-0-1-55-deprecations"]
+=== Deprecations
+
+* Specifying "Install into all namespaces in the cluster" or setting the `WATCH_NAMESPACES` environment variable to `""` no longer affects all namespaces. This change improves the Compliance Operator's memory usage.
```

Comment:
I think we should expand on this. The effect on the end user is that any API resources installed in other namespaces than the one specified for installation will not work anymore. Typically this would mean that all resources would have to be created in the openshfit-compliance namespace.